### DISCUSSION TITLE
Can I apply for Github Campus Expert? I am in my last phase of College(Almost Graduated).

### DISCUSSION THREAD

**Discussion by devarsh10** [community/community#128216] [GitHub Education] [answered]
### Hi everyone,

I am in my final year of college and have completed all my university exams. I have enjoyed leading sessions on Cloud and DevOps and spreading knowledge to others. While I am afraid I may not be eligible for the GitHub Campus Expert Program due to being in my final year, I'm still interested in being a Campus Expert or finding a similar way to contribute.

Please help

Warm Regards,
Devarsh Shah.

**Comment by MetzinAround** [comment, marked as answer]
Hi @devarsh! thanks for the question. 
For GitHub Campus Experts, we require at least 1 year of school left when applying. Our next round of applications is August 2024, so if you are graduating before August 2025, then you would be ineligible to become a Campus Expert. 
I'm sorry to bring this news to you, but I am excited to hear that you have been organizing so many ways to help your community to learn and grow. I encourage you to see [if there are any Campus Experts in your area](https://githubcampus.expert/experts) to see if they want help organizing (many of them organize events for more than just their campus). 

Have a great day and end to your school year! Congrats on finishing your exams!

**Reply by Jasonradford51** [reply]
i have no idea pal im in work  so i can barely touch my computer but but ah well

**Comment by devarsh10** [comment]
Hey @MetzinAround,
Thank you so much for your reponse.
I will follow the guidance you gave.

Thanks and Regards

**Comment by huda-code** [comment]
I am a graduate (Master's) student, I will be graduating in Dec 2025, Can I apply for Aug 2024's GitHub Campus expert program?

**Reply by MetzinAround** [reply]
Yes you can!
